contracts/PriceOracle.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.5.16;
```

Comment:
Oh, compiler versions come into play 😒

Our core is on ^0.5.16, while other repos use ^0.8. Let's change this to ^0.8.0 and see if hardhat can deal with multiple compiler versions in the core repo?

Reply:
ok let test